### DISCUSSION TITLE
[client-chime-sdk-media-pipelines] retrieving info about completed media pipelines older 5-10 days

### DISCUSSION THREAD

**Discussion by golubevk** [aws/aws-sdk-js-v3#5394] [General]
### Describe the issue

According using the GetMediaPipeline method I can only get an existing media pipeline.

Is it possible to retrieving info about completed media pipelines older than 5-10 days?

### Links
https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-client-chime-sdk-media-pipelines/
https://docs.aws.amazon.com/chime-sdk/latest/APIReference/API_media-pipelines-chime_GetMediaPipeline.html

**Comment by RanVaknin** [comment]
Hi @golubevk,

Thanks for reaching out. From looking at the API reference and docs of media pipeline I did not find a way to get information about a media pipeline that doesn't exist anymore.  I don't see any references in the docs for "completed" media pipelines, and I think it's intentional. Its either that the media pipeline exists or it doesn't.

Thanks,
Ran~

**Reply by golubevk** [reply]
Hi, @RanVaknin Thanks for answering!

Let's assume the pipeline was created with the Failed status. How long will the pipeline exist in the system before the GetMediaPipeline API starts responding with a NotFoundException error?
